catalog/video/video-admin-plugin/pom.xml

```diff
@@ -71,12 +68,116 @@
             <artifactId>powermock-module-junit4-rule-agent</artifactId>
             <version>${powermock.version}</version>
         </dependency>
-
+        <dependency>
+            <groupId>org.codice.webjars</groupId>
+            <artifactId>backbone.modelbinder</artifactId>
+            <version>1.1.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.webjars.bower</groupId>
+            <artifactId>bootswatch</artifactId>
+            <version>3.2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.webjars.bower</groupId>
+            <artifactId>components-backbone</artifactId>
+            <version>1.1.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.webjars.bower</groupId>
+            <artifactId>components-bootstrap</artifactId>
+            <version>3.2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>org.webjars.bower</groupId>
+            <artifactId>components-font-awesome</artifactId>
```

Comment:
since this is used in 3 places, it may be beneficial to add it to dependencyManagement as well